### ISSUE TITLE
upload to deken

### ISSUE THREAD

**Issue by umlaeute** [iainctduncan/scheme-for-pd#22]
binaries should be uploaded to `deken`, so it's easy to install s4pd from within Pd.

**Comment by iainctduncan** [comment]
Yes, this will be done, but only after I have successful windows installs happening.
